app/components/land-use-map.js

```diff
@@ -1,12 +1,110 @@
 import Ember from 'ember'; // eslint-disable-line
 
 export default Ember.Component.extend({
-  geomStyle: {
-    fill: false,
-    color: '#000',
-    weight: 2,
-    dashArray: '8 5',
-    opacity: 0.6,
+  initOptions: {
+    style: 'mapbox://styles/mapbox/light-v9',
+    zoom: 9,
+    center: [-74, 40.7071],
+    scrollZoom: false,
   },
+
+  vectorSource: Ember.computed('mapState.landUseTemplate', function () {
+    return {
+      type: 'vector',
+      tiles: ['https://tiles.planninglabs.nyc/pluto/{z}/{x}/{y}/tile.mvt'],
+      minzoom: 14,
+    };
+  }),
+
+  rasterSource: Ember.computed('mapState.landUseTemplate', function () {
```

Comment:
I don't think these need to be computed properties if they're not doing anything aside from returning object literals.